src/mpi/coll/transports/gentran/gentran_utils.c

```diff
@@ -341,17 +341,17 @@ int MPII_Genutil_progress_hook(int *made_progress)
     in_genutil_progress = 1;
 
     if (made_progress)
-        *made_progress = FALSE;
+        *made_progress = false;
```

Comment:
Please revise commit messages (` coll/tsp: only count towards PROGRESS_MAX_COLLS when actually makes p…

…rogress`). The short message breaking up is no good. It didn't explain the addition of local progress variable (no matter how obvious it appears at the time).

Reply:
I am not sure I get it. But I edited the commit message in the other PR.